characters_counter/characters_counter.py

```diff
@@ -0,0 +1,20 @@
+from collections import Counter
+from functools import cache
+
+REQUIREMENT_OCCURRENCE = 1
+
+
+@cache
+def count_characters_occurrence(string: str) -> int:
+    """
+    The function returns the number of characters in the string occuring only
+    once.
+    """
+    characters_counts = Counter(string)
+    unique_occurrences = filter(
```

Comment:
![image](https://github.com/user-attachments/assets/89a14524-ba09-479f-8033-0ace624182b9)
Please fix the formatting
There is no reason of having that whole block indented that far
https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html#reformat_file

Pay attention to the opening and closing parentheses
Here are few examples

1. You open the parentheses and keep code in the same line
```
variable = dict(var=1)
```
That is the standard way

2.  You open the parentheses and keep code in the same line but you have multiple lines then
![image](https://github.com/user-attachments/assets/1c678d5f-f27b-4648-a542-c9710a626e44)

See how the indentation is preserved and both parentheses stay near the code. So you have open one near start of the code and closing one near the code

3. You open the parentheses and keep code at the new lines (your variant):
```
variable = dict(
    var=1,
    var=2
)
```
See how this time we moved closing one at the new line, that's because the opening one didn't have the code near it.
You can also do the following:
```
variable = dict(
    var=1, var=2
)
```
I don't mind how you format your code as long as it readable and you follow this standard https://realpython.com/python-pep8/#:~:text=PEP%208%20suggests%20lines%20should,to%20run%20over%20several%20lines.

Reply:
It seems I understand, but sometimes there are situations with indentations and parenthesis that baffled me. That was one of them. Your links cleared things up a bit.